npm run build emits TypeError for undefined Properties

**Bug description**
When running `npm run build` an error is emitted.

```
$ npm run build

> react-lazy-load-image-component@1.1.5 build /Users/rpierce/source/GithHub/react-lazy-load-image-component
> webpack

/Users/rpierce/source/GithHub/react-lazy-load-image-component/node_modules/webpack-cli/bin/config-yargs.js:89
				describe: optionsSchema.definitions.output.properties.path.description,
				                                          ^

TypeError: Cannot read property 'properties' of undefined
    at module.exports (/Users/rpierce/source/GithHub/react-lazy-load-image-component/node_modules/webpack-cli/bin/config-yargs.js:89:47)
    at /Users/rpierce/source/GithHub/react-lazy-load-image-component/node_modules/webpack-cli/bin/webpack.js:60:27
    at Object.<anonymous> (/Users/rpierce/source/GithHub/react-lazy-load-image-component/node_modules/webpack-cli/bin/webpack.js:515:3)
    at Module._compile (module.js:577:32)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-lazy-load-image-component@1.1.5 build: `webpack`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-lazy-load-image-component@1.1.5 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/rpierce/.npm/_logs/2018-12-13T04_18_14_888Z-debug.log
```

**To Reproduce**
Paste the code that is producing the issue
```
npm run build
```

List any other actions needed to reproduce the issue:
- `rm -rf node_modules && npm i`

Link to a repo where this issue can be reproduced if available:

**Expected behavior**
`npm run build` succeeds without error

**Technical details:**
- This seems to be a [known issue](https://github.com/webpack/webpack-cli/issues/607) with the version of webpack-cli@2
- Upgrading to webpack-cli@3 seems to fix the issue and the build passes.

--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
     "react-dom": "^16.2.0",
     "style-loader": "^0.20.3",
     "webpack": "^4.17.1",
-    "webpack-cli": "^2.0.12"
+    "webpack-cli": "^3.1.2"
   },
   "scripts": {
     "test": "jest",

